src/libNode/Node.cpp

```diff
@@ -60,6 +60,39 @@ const unsigned int MIN_CHILD_CLUSTER_SIZE = 2;
 
 #define IP_MAPPING_FILE_NAME "ipMapping.xml"
 
+#ifndef PRODUCTION_BUILD
+namespace {
+// testing version
+[[maybe_unused]] int readAccountJsonFromFile(const string& path) {
```

Comment:
I suggest using std::filesystem::path for paths.

Reply:
Again if we find this useful then we should clean it up, its from our own code base.